orchagent/switchorch.cpp

```diff
@@ -52,16 +55,32 @@ void SwitchOrch::doTask(Consumer &consumer)
                 }
 
                 auto value = fvValue(i);
-                if (packet_action_map.find(value) == packet_action_map.end())
-                {
-                    SWSS_LOG_ERROR("Unsupported packet action %s", value.c_str());
-                    it = consumer.m_toSync.erase(it);
-                    continue;
-                }
 
                 sai_attribute_t attr;
                 attr.id = switch_attribute_map.at(attribute);
-                attr.value.s32 = packet_action_map.at(value);
+
+                switch (attr.id)
+                {
+                    case SAI_SWITCH_ATTR_FDB_UNICAST_MISS_PACKET_ACTION:
+                    case SAI_SWITCH_ATTR_FDB_BROADCAST_MISS_PACKET_ACTION:
+                    case SAI_SWITCH_ATTR_FDB_MULTICAST_MISS_PACKET_ACTION:
+                        if (packet_action_map.find(value) == packet_action_map.end())
+                        {
+                            SWSS_LOG_ERROR("Unsupported packet action %s", value.c_str());
+                            it = consumer.m_toSync.erase(it);
+                            continue;
+                        }
+                        attr.value.s32 = packet_action_map.at(value);
+                        break;
+
+                    case SAI_SWITCH_ATTR_ECMP_DEFAULT_HASH_SEED:
+                    case SAI_SWITCH_ATTR_LAG_DEFAULT_HASH_SEED:
+                        attr.value.u32 = to_uint<uint32_t>(value);
```

Comment:
do we have test cases for different hash seeds?

Reply:
No, as far as I know, we don't have.